app/assets/stylesheets/mtl/extend/_forms.scss

```diff
@@ -17,3 +17,37 @@
 .input-field.col label {
   padding-right: $gutter-width;
 }
+
+// fields with text suffix/unit
+.input-field.has-suffix {
+  input:not([type]),
```

Comment:
I'm not a big fan either. Nonetheless I think we should stick to the same selector structure as the base library.